lib/image_edit_screen.dart

```diff
@@ -0,0 +1,219 @@
+import 'dart:io';
+
+import 'package:flutter/foundation.dart' show kIsWeb;
+import 'package:flutter/material.dart';
+import 'package:crop/crop.dart';
+
+class ImageEditScreen extends StatefulWidget {
+  final String imagePath;
+
+  const ImageEditScreen({super.key, required this.imagePath});
+
+  @override
+  State<ImageEditScreen> createState() => _ImageEditScreenState();
+}
+
+class _ImageEditScreenState extends State<ImageEditScreen> {
+  final controller = CropController(aspectRatio: 9.0 / 16.0);
+  double rotation = 0;
+  BoxShape shape = BoxShape.rectangle;
+
+  RawImage? croppedImage;
+  File? originaFile;
+
+  void initState() {
+    super.initState();
+  }
+
+  Future<void> cropImage() async {
+    final pixelRatio = MediaQuery.of(context).devicePixelRatio;
+    final cropped = await controller.crop(pixelRatio: pixelRatio);
+
```

Comment:
VS Code gives this warning: Do not use BuildContexts across async gaps.

Make sure you add this code before the Navigator.of(context).push() (Line 32)

```suggestion
if (!context.mounted) return;
```

Reply:
context.mounted errors out on my vs code when I try to add that line. Are you sure you mean context.mounted and not  just mounted by itself? Since that's available.